src/libcmis/gdrive-document.cxx

```diff
@@ -69,35 +70,48 @@ string GDriveDocument::getDownloadUrl( string streamId )
     if ( !streamId.empty( ) )
     {
         // Find the rendition associated with the streamId
-        for ( vector< RenditionPtr >::iterator it = renditions.begin( ) ; 
-            it != renditions.end(); ++it )
+        std::vector<RenditionPtr>::iterator it = std::find_if(renditions.begin(), renditions.end(),
```

Comment:
TBH, I don't think this improves the code in any way... If you really want to touch this, maybe using foreach loops would be better? That should make the code easier to read.

Reply:
Just to be sure, before I do anything wrong, do you prefer a foreach for the 4 changes or only for this one?

Reply:
For all of them.

Reply:
Ok done.